Anchor footer to content area. Affix side nav. closes #165

webinista opened the pull request:
- Makes side navigation fixed
- Makes header fixed on internal pages
- Adds left margin to footer on internal pages.
  - Adds conditional that checks whether we're on home page or internal page and adds a class name to the footer.
  - Adds .va-rr-footer--alt class
- Moved variables from _main-content.scss to _va-rr-vars.scss and moved _va-rr-footer.scss to components/ directory so that menu variables work are available repo-wide.

webinista committed:
Anchor footer to content area. Affix side nav. closes #165

- Makes side navigation fixed
- Makes header fixed on internal pages
- Adds left margin to footer on internal pages.
	- Adds conditional that checks whether we're on home page or internal page and adds a class name to the footer.
	- Adds .va-rr-footer--alt class
- Moved variables from _main-content.scss to _va-rr-vars.scss and moved _va-rr-footer.scss to components/ directory so that menu variables work are available repo-wide.

webinista committed:
Anchor footer to content area. Affix side nav. closes #165

- Makes side navigation fixed
- Makes header fixed on internal pages
- Adds left margin to footer on internal pages.
	- Adds conditional that checks whether we're on home page or internal page and adds a class name to the footer.
	- Adds .va-rr-footer--alt class
- Moved variables from _main-content.scss to _va-rr-vars.scss and moved _va-rr-footer.scss to components/ directory so that menu variables work are available repo-wide.

webinista committed:
Anchor footer to content area. Affix side nav. closes #165

- Makes side navigation fixed
- Makes header fixed on internal pages
- Adds left margin to footer on internal pages.
	- Adds conditional that checks whether we're on home page or internal page and adds a class name to the footer.
	- Adds .va-rr-footer--alt class
- Moved variables from _main-content.scss to _va-rr-vars.scss and moved _va-rr-footer.scss to components/ directory so that menu variables work are available repo-wide.

emilyville commented:
:+1: !! Looks  good @webinista 

you didn't assign it so not merging (since I'm not sure you're done?) but if you are done go ahead and merge.

Commit:
merge with master

emilyville commented:
This isn't playing nice with the new beta thing on the header!

![screen shot 2016-03-14 at 6 48 39 pm](https://cloud.githubusercontent.com/assets/1307774/13762569/3a2e84c6-ea16-11e5-8d88-dd054eb6ac06.png)

webinista committed:
Fixing beta notice layout issue.

modified:   _sass/components/_main-content.scss
modified:   _sass/components/_va-rr-header.scss
modified:   css/main.scss

webinista committed:
Fixing card title regression. closes #175

Also does the following:
- Changing h4 font-family to Roboto Slab.
- Making specificity of .va-card-title 2,0,0.
- Cleans out unused CSS in _main-content.scss.
- Changes heading hierarchy for files in _design collection.

Commit:
Merge pull request #177 from department-of-veterans-affairs/issue#175

Fixing card title regression. closes #175

Commit:
delete key section in side nav

thecapacity committed:
Update Rakefile - minor typos

s/extenral/external/

Commit:
Merge pull request #183 from thecapacity/patch-1

Update Rakefile - minor typos

webinista committed:
Making header compatible with new beta indicator.

webinista committed:
Merge branch 'issue#165' of github.com:department-of-veterans-affairs/roadrunner
